libraries/aws_iam_role.rb

```diff
@@ -45,4 +55,21 @@ def exists?
   def to_s
     "AWS IAM Role #{@role_name}"
   end
+
+  private
+
+  def fetch_attached_role_policies(role_name)
+    catch_aws_errors do
+      resp = @aws.iam_client.list_attached_role_policies({ role_name: role_name }).attached_policies
+      @attached_policies_names = []
+      @attached_policies_arns  = []
+      unless resp.empty?
+        resp.each do |r|
+          @attached_policies_names << r.policy_name
+          @attached_policies_arns  << r.policy_arn
+        end
+      end
+      return @attached_policies_names && @attached_policies_arns
```

Comment:
Do you need to return anything from this method? Can just add a bang `!` to the method and return nothing, as it updates `attached_policies_names` and `attached_policies_arns` in-place.

Reply:
Ah meant to remove this return. Was re-writing this method for lazy loading, but changed my mind.